lib/modules/dosomething/dosomething_reportback/dosomething_reportback.admin.inc

```diff
@@ -169,9 +169,7 @@ function dosomething_reportback_files_form($form, &$form_state, $status = NULL,
 
   foreach ($result as $record) {
     $rb_path = 'admin/reportback/' . $record->rbid;
-    // @TODO: temporarily disable pre-approved to unblock campaigns team from reviewing reportbacks.
-    // Refs #4797
-    // $status = 'approved';
+    $status = 'approved';
```

Comment:
@angaither @weerd does it make sense to hold off on adding back approve as default selected until we verify that we're no longer getting the illegal choice detected warning?